### ISSUE TITLE
Spells and Potions are not working

### ISSUE THREAD

**Issue by whoedson** [Azami7/Ollivanders2#511]
I've been using this plugin for a while and every since the latest update lots of spells and potions are not working:

- Basically all transfigurations spells (spongify, duro, all incarnatio, etc...)
- Pyrosvestiras
- Immobulus (basically no slow is applied)
- Defodio (only breaks 1 block)
- Protego Horribilis
- Horreat Protegat 

Maybe there are more spells that are broken, havent test them all

Potions are also not giving any effect.

**Comment by whoedson** [comment]
My server its in 1.17.1

**Comment by Azami7** [comment]
taking a look

**Comment by Azami7** [comment]
I am not able to reproduce any issues with the stationary spells but my version of the plugin has many changes from what is on Spigot so it is possible something we changed previously fixed an issue.

Potions, Pyrosvestiras - fixed in next release
Other transfiguration - we only know of issues with Spongify and Aguamenti, both have been fixed in the next release
Defodio - something has changed in 1.17 in how blocks break which is confusion this spell. I can get it to break 2-3 blocks but not more. Will need to investigate further.
